tasks/main.yml

```diff
@@ -254,18 +252,27 @@
     mode: "750"
   when: item
   with_items:
-    - "{{ paperlessng_consumption_dir }}"
     - "{{ paperlessng_data_dir }}"
     - "{{ paperlessng_trash_dir }}"
     - "{{ paperlessng_media_root }}"
     - "{{ paperlessng_staticdir }}"
+    - "{{ paperlessng_export_dir }}"
+
+- name: create paperless-ngx consumption directory and set permissions
+  file:
+    path: "{{ paperlessng_consumption_dir }}"
+    state: directory
+    owner: "{{ paperlessng_system_user }}"
+    group: "{{ paperlessng_system_group }}"
+    mode: "750"
+  when: consumption_mount_exist == false
 
 - name: rename initial config
   command:
     cmd: "mv -f {{ paperlessng_directory }}/paperless.conf {{ paperlessng_directory }}/paperless.conf.template"
     removes: "{{ paperlessng_directory }}/paperless.conf"
 
-- name: configure paperless-ng
+- name: configure paperless-ngx
```

Comment:
This fails with the following vars undefined:
- paperlessng_url
- paperlessng_enable_update_check

Reply:
Added both into defaults, missed out initially as was part of my group_vars.